.github/workflows/ci-docs-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+name: ci-docs-workflow
+run-name: Generating docs for ${{ github.ref }}
+on:
+  push:
+    branches:
+      - main
+      - dev
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install deps
+        run: npm i
+
+      - name: Generate docs
+        run: npx oclif readme
+
+      - name: Commit docs
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git commit -am "Generate docs: ${{ github.ref }}"
+          git push
```

Comment:
What are we pushing if the `ReadMe` is already in repo?

Reply:
Since OCLIF deterministically generates docs for the commands, manual modifications made to the OCLIF controlled segments get overwritten.

However, all other changes to the readme are respected.